### PR TITLE
python 3 conversion

### PR DESCRIPTION
**Pull request by golnazads**

### CODE REVIEW

File: authoraffsrv/views.py [hunk 1]
```diff
@@ -138,7 +142,7 @@ def __export_to_excel(self):
         filename = self.TMP_EXCEL_FOLDER + self.EXPORT_FILENAME + str(uuid.uuid4())
         wbk.save(filename)
         # now read it
-        xls_file = open(filename)
+        xls_file = open(filename, encoding='ISO-8859-1')
```
Comment (thostetler) [inline authoraffsrv/views.py:145]: I'm not super familiar, but any reason not to use UTF-8? 
I think 8859 is pretty old if I'm not mistaken.

**Review by thostetler — APPROVED** [review]
LGTM

**Comment by golnazads** [comment]
thank you so much Tim. utf-8 did not work. someone had suggested that so I
used it and tests passed. I want to try it live and see if it is OK. I
shall try searching again and see what else I can find. thank you.

File: authoraffsrv/views.py (hunk 1 above)
Comment (golnazads) [reply authoraffsrv/views.py:145]: as per your suggestion removed the encoding. apparently if you open it in byte mode you do not need the encoding. going to see if it works live. thank you for your suggestion.

**Pull request merged**